src/normals.c

```diff
@@ -114,7 +114,10 @@ stl_fix_normal_directions(stl_file *stl) {
   tail->next = tail;
 
   /* Initialize list that keeps track of already fixed facets. */
-  norm_sw = (char*)calloc(stl->stats.number_of_facets, sizeof(char));
+  const size_t number_of_facets = stl->stats.number_of_facets;
```

Comment:
number_of_facets is declared as int. why puting it into size_t instead?

Reply:
The code does not need a local copy of this value.

Reply:
OK, thanks, I will change to int. Local copy will hopefully not be created due to compiler optimizations. But for readability, it should be better.